mock/azure.go

```diff
@@ -1,4 +1,210 @@
 package mock
 
 // AzureHealthy is a JSON string representing a healthy Azure cluster
-var AzureHealthy = `{}`
+var AzureHealthy = `
```

Comment:
Does this mock get used right away now?

Reply:
Yes, all wired up! Checked by temporarily removing nodes and getting `red` health as expected.